Mathlib/RingTheory/NonUnitalSubring/Basic.lean

```diff
@@ -800,6 +800,14 @@ theorem map_iSup {ι : Sort*} (f : F) (s : ι → NonUnitalSubring R) :
     (iSup s).map f = ⨆ i, (s i).map f :=
   (gc_map_comap f).l_iSup
 
+theorem map_inf (s t : NonUnitalSubring R) (f : F) (hf : Function.Injective f) :
+    (s ⊓ t).map f = s.map f ⊓ t.map f := SetLike.coe_injective (Set.image_inter hf)
+
+theorem map_iInf {ι : Sort*} [Nonempty ι] (f : F) (hf : Function.Injective f)
+    (s : ι → NonUnitalSubring R) : (iInf s).map f = ⨅ i, (s i).map f := by
```

Comment:
let's not mix notations, and we generally prefer `map` beforehand, despite dot notation. Can you please change these issues here and throughout?
```suggestion
    (s : ι → NonUnitalSubring R) : map f (⨅ i, s i) = ⨅ i, map f (s i) := by
```

Reply:
I'm not sure about this. Since the existing part of this file already uses dot notation, I think it's OK to follow these styles?

Reply:
Fair enough. It's not a big deal either way.